mpdsync.py

```diff
@@ -91,9 +94,9 @@ def main():
         sync(master, slaves)
         
     
-def get_settings(settings_file="settings.json"):
+def get_settings(settings_file=os.path.join(BaseDirectory.xdg_config_home, "mpdsync.json")):
```

Comment:
Yeah, I wondered if it was a good idea to depend on python-xdg, and if it was really necessary.  I doubt that many people actually change $XDG_CONFIG_HOME to something other than `~/.config`.  On the other hand, python-xdg looks to be a fairly standard package (especially on Ubuntu, where software-center depends on it, so it should be installed by default).  And it seems like the more "correct" way to do it, rather than hard-coding paths.

But I don't really care very much; I use `~/.config` anyway.  I suppose the best reason might be to make it easier for potential Windows users, but I don't care much about supporting Windows either, haha!

So, yeah, if you prefer doing it that way, that's fine with me.  Thanks.